deployment-templates/deploy-forseti-client.yaml.in

```diff
@@ -46,20 +46,10 @@ resources:
     gcs-bucket: {FORSETI_BUCKET}
 
     # --- Forseti version
-    # Use either branch-name or release-version, but NOT both.
+    # "forseti-target": Can be a tag or a branch.
```

Comment:
Rather than using `target`, I think that the code would be more clear and easy to read if we use `version` instead.  How do think?

Reply:
Updated.